src/Nri/Ui/Block/V6.elm

```diff
@@ -542,7 +542,7 @@ toMark config { backgroundColor, borderColor } =
                     Underline ->
                         Css.solid
 
-            notBottomBorderColor =
+            borderColor_ =
```

Comment:
Do we definitely want to use a different color for the underline than the text?

<img width="391" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/5c0d3000-45c1-40ca-8252-1ca2f6f24338">

Reply:
Ahh I see! This is happening because of the user agent stylesheet.

In the screenshot above, there's a label, so the blank is treated as highlighted by "fruit." Semantically, we're using the `mark` tag for this:
<img width="501" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/1299f744-a78b-48e3-b4de-3c27b4980f7e">

If there's no label or emphasis, the styles apply as expected:
<img width="310" alt="image" src="https://github.com/NoRedInk/noredink-ui/assets/8811312/e36e2608-bf32-456e-9abc-090642a9d023">

(I don't have an opinion on whether this is worth addressing as part of this PR, but if you do override the user agent stylesheet, please check that high contrast mode keeps working.)